packages/app-project/stores/User/UserPersonalization/Notifications/Notifications.js

```diff
@@ -0,0 +1,85 @@
+import { flow, getRoot, types } from 'mobx-state-tree'
+import { sugarClient } from 'panoptes-client/lib/sugar'
+import talkClient from 'panoptes-client/lib/talk-client'
+import asyncStates from '@zooniverse/async-states'
+
+const Notifications = types
+  .model('Notifications', {
+    count: types.maybeNull(types.number),
+    error: types.maybeNull(types.frozen({})),
+    loadingState: types.optional(
+      types.enumeration('state', asyncStates.values),
+      asyncStates.initialized
+    )
+  })
+  .actions(self => {
+    return {
+      fetchAndSubscribe () {
+        self.fetchInitialUnreadNotifications()
+        self.subscribeToSugarNotifications()
+      },
+
+      fetchInitialUnreadNotifications: flow(function * fetchInitialUnreadNotifications () {
+        self.setLoadingState(asyncStates.loading)
+        try {
+          const query = {
+            delivered: false,
+            page_size: 1
+          }
+          const response = yield talkClient.type('notifications').get(query)
+          const [notification] = response
+          self.count = notification.getMeta().count
+
+          self.setLoadingState(asyncStates.success)
+        } catch (error) {
+          self.handleError(error)
+        }
+      }),
+
+      processSugarNotification (notification) {
+        // sugar data objects of source_type = 'Message' are related to Talk Conversations and the nav item "Messages", not the nav item "Notifications"
+        // TODO: add nav item "Messages" count
+
+        if (notification && notification.data.source_type !== 'Message') {
```

Comment:
```suggestion
        if (notification?.data.source_type !== 'Message') {
```

Reply:
It's been ages since I worked with Sugar, but will this fire when other notifications come in, like intervention messages? We aren't using those at the moment, but they're worth bearing in mind for future work.

Reply:
Answering my own question, interventions used `message.type: 'experiment'` but Talk uses `message.type: 'notification'` so they shouldn't clash.

Reply:
I've updated my wip with the conditional chaining suggestion, thanks!

That's correct regarding the message types. This PR only subscribes to the Talk related sugar messages, which I think can have `source_type` of:
- Comment - should increment header unread Notifications count
- Discussion - should increment header unread Notifications count
- Message - should increment header unread Messages count if related conversation not already included in count
- Moderation (just figured this one out, which shouldn't increment Notifications or Messages count, I'll update code accordingly)